### DISCUSSION TITLE
Embedchain cookbooks

### DISCUSSION THREAD

**Discussion by deshraj** [mem0ai/mem0#824] [Announcements]
Hello, everyone! 🚀

We're thrilled to announce some fantastic updates!

We've just loaded up a treasure trove of examples, suitable for both Google Colab and REPLs, showcasing the incredible versatility of Embedchain with various Language Models (LLMs), embedding models, and vector databases. Check out the extensive list in our GitHub README: https://github.com/embedchain/embedchain#examples ❤️

Got any burning questions or want to share some thoughts? Don't be shy – we'd love to hear from you! 📣

Thank you for your support and enthusiasm!

Cheers,
Team Embedchain 🌟

**Comment by MaksimZinovev** [comment]
Thank you for examples! Is there a guide somewhere on how to deploy API server to cloud?

**Reply by deshraj** [reply]
We have an example here on how to build REST API service using Embedchain: https://docs.embedchain.ai/examples/api_server

We don't have instructions on how to deploy to a cloud provider but you can do that easily by just pulling the docker image and running the container.
